erpnext/accounts/report/marketing_expense/marketing_expense.json

```diff
@@ -0,0 +1,28 @@
+{
+ "add_total_row": 1,
+ "creation": "2020-08-08 20:17:15.211118",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "modified": "2020-08-09 00:44:02.499574",
+ "modified_by": "Administrator",
+ "module": "Accounts",
+ "name": "Marketing Expense",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "query": "SELECT\n    name as \"ID:Link/Sales Invoice:100\",\n    posting_date as \"Date::100\",\n    customer as \"Customer:Link/Customer:150\",\n    company as \"Company::180\",\n    order_type as \"Order Type::120\",\n    discount_amount as \"Marketing Expense::120\",\n    Status as \"status::200\"\nFROM\n    `tabSales Invoice`\nWHERE\n    order_type = \"Marketing\"\n    AND company = %(company)s\n    AND posting_date BETWEEN %(from_date)s AND %(to_date)s\n",
```

Comment:
Don't pull the Order Type field in the report, it will always be Marketing anyway